backend/app/routes/bookings/booking_service.py

```
@@ -0,0 +1,78 @@
+        await db.refresh(job)
+        return job
+
+    @staticmethod
+    async def get_all_jobs(db: AsyncSession):
+        """
+        Fetch all rows from the detailing_service table.
+        
+        Returns:
+            A list of rows representing every record in the `detailing_service` table.
+        """
+        from sqlalchemy import select
```

Ruff (0.14.10)

Redefinition of unused `select` from line 2

Remove definition: `select`

(F811)